h/static/styles/partials-v2/_search.scss

```diff
@@ -181,6 +207,15 @@
 @media screen and (max-width: $tablet-width + 225px) {
   .search-result-container {
     flex-direction: column;
+    margin-top: 25px;
+  }
+
+  .search-result-nav {
+    display: flex;
+  }
+
+  .hide-on-small-screens {
```

Comment:
Hmm ... this class doesn't follow our naming conventions - where every class is defined in a file that starts with a prefix of the class name.